config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
   resources :thoughts
-  get 'homepage/greeting'
+  root to: 'homepage#index'
```

Comment:
This change makes it so the home page is served at root `http://localhost:3000/`